examples/Receiver_Response.py

```diff
@@ -0,0 +1,164 @@
+""" Calculate and Fit Receiver Response to Sinc
+===============================================
+
+Two files are required for the following example:
+
+    File1 contains the analytic signal acquired on the GDS oscilloscope
+    directly output from the AFG output. Each node pertains to signal with a
+    different frequency (in kHz) which are fit to a complex function to extract
+    the power (in :math:`V^{2}`).
+    File2 contains the quadrature signal acquired on the receiver when the same
+    signal of File1 is injected into it. Each node pertains to signal with a
+    different frequency (in kHz) which are converted to a PSD. The power (in
+    :math:`dg^{2}`) is calculated from the peak of the convolved PSD.
+
+    The receiver response is then the ratio of :math:`dg^{2}(\\nu)` to
+    :math:`V^{2}(\\nu)` which is fit to a sinc function. The final plot shows
+    the square of the data with the fit.
+"""
+import numpy as np
+import pyspecdata as psd
+from sympy import symbols
+import sympy as sp
+import re
+
+
+def for_plot(thisdata):
+    """Converts :math:`dg^{2}/V^{2}` to :math:`dg/\\mu V`
+    and sets labels accordingly for pretty plotting"""
+    # Take sqrt for units of $dg/V$
+    thisdata.data = np.sqrt(thisdata.data)
+    # Units of $dg/\mu V$
+    thisdata /= 1e6
+    thisdata.rename("nu_test", r"$\Delta \nu$")
+    thisdata.name(r"$\mathrm{dg_{%s\ \mathrm{kHz}}}/ \mu V$" % SW)
+    return thisdata
+
+
+file1 = "240123_10mV_AFG_GDS_5mV_100MSPS_analytic.h5"
+file2 = "240117_afg_sc_10mV_3p9kHz_zoom.h5"
+lambda_G = 0.4e3  # Width for Gaussian convolution
+# There are less nodes acquired for the control case (since we assume it's
+# relatively flat) so I need to separately define them
+file1_nodes = psd.find_file(
+    re.escape(file1),
+    exp_type="ODNP_NMR_comp/noise_tests",
+    return_list=True,
+)
+control_frqs_kHz = np.array([float(j.split("_")[1]) for j in file1_nodes])
+file2_nodes = psd.find_file(
+    re.escape(file2),
+    exp_type="ODNP_NMR_comp/noise_tests",
+    return_list=True,
+)
+# $\nu_{test}$
+resp_frq_kHz = np.array([float(j.split("_")[1]) for j in file2_nodes])
+# {{{ Calculate input power (acquired on Oscilloscope)
+#     To determine the input power, just take a capture of the signal in the
+#     time domain and fit to a complex exponential
+# {{{ Make empty nddata to drop the calculated $V^{2}$ into with corresponding
+#     frequency ($\nu$) output by AFG source, and set the frequencies based on
+#     the node names
+control = psd.ndshape([len(control_frqs_kHz)], ["nu_test"]).alloc()
+control.setaxis("nu_test", control_frqs_kHz).set_units("nu_test", "Hz")
+# }}}
+for j, nodename in enumerate(file1_nodes):
+    rf_frq = control["nu_test"][j]
+    d = psd.find_file(
+        file1,
+        expno=nodename,
+        exp_type="ODNP_NMR_comp/noise_tests",
+    )
```

Comment:
note I'm not using the re which you had started to use. This way is a. more consistent with the process_gain example and b. uses less lines of code also I find it's easier to follow